gym-unity/gym_unity/envs/__init__.py

```diff
@@ -18,9 +18,8 @@ class UnityGymException(error.Error):
     pass
 
 
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger("gym_unity")
-
+logger = get_logger(__name__)
+set_log_level(INFO)
```

Comment:
Very nit: it seems kind of weird to import `INFO` as a global variable - could we import logging_util and do `logging_util.INFO` as we do in the `learn.py`?

Reply:
Sure, sounds good.